Task5/ArrayHelper.js

```diff
@@ -0,0 +1,63 @@
+// 1-https://www.codewars.com/kata/array-helpers
+Array.prototype.square = function () {
+    var givenArr = this;
+    var squareArr = [];
+    givenArr.forEach(squareElement);
+    function squareElement(element, index) {
+        squareArr[index] = Math.pow(element, 2);
+    }
```

Comment:
you can write the above lines as below:

givenArr.forEach((element, index) => {
         squareArr[index] = Math.pow(element, 2);
     });

no need to create a separate method and then link it,
its not necessary to separate them you can place them in same lines as written above.

you can refactor the below prototype methods also